lib/host_kafka.py

```diff
@@ -0,0 +1,35 @@
+import socket
+from contextlib import closing
+
+from app.config import Config
+from app.environment import RuntimeEnvironment
+from app.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def _any_bootstrap_server_connects(kafka_socket, servers) -> bool:
+    if not servers:
+        config = Config(RuntimeEnvironment.SERVICE)
+        servers = [config.bootstrap_servers] if isinstance(config.bootstrap_servers, str) else config.bootstrap_servers
+
+    try:
+        for server in servers:
+            try:
+                host, port = server.split(":")
+                new_addr = (host, int(port))
+
+                # connect_ex() returns zero when the socket is open and accessible
+                errcode = kafka_socket.connect_ex(new_addr)
+                if not errcode:
```

Comment:
For clarity and to avoid any unlikely type coercion issues. Maybe also make 0 a constant in this file for more clarity. Then you could remove the comment

```suggestion
                if errcode == SUCCESS_CODE:
```

Reply:
Now using `errcode == 0`